ecommerce/android/Getflix/app/src/main/res/layout/order_product_item.xml

```diff
@@ -0,0 +1,87 @@
+            type="com.example.getflix.models.OrderPurchasedModel" />
+    </data>
+
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="120dp"
+        android:elevation="3dp"
+        android:padding="1dp">
+
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/cart_product_image"
+                android:layout_width="96dp"
+                android:layout_height="match_parent"
+                android:layout_marginStart="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="8dp"
+                android:paddingTop="4dp"
+                android:src="@drawable/zara_jacket3"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/cart_product_name"
+                android:layout_width="180dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_marginTop="16dp"
+                android:fontFamily="sans-serif-condensed"
+                android:gravity="left"
+                android:paddingTop="3dp"
+                android:text="@{listproduct.product.name}"
+                android:textColor="@color/black"
+                android:textSize="14sp"
+                android:textStyle="bold"
+                app:layout_constraintStart_toEndOf="@+id/cart_product_image"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/cart_product_price"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_marginTop="8dp"
+                android:fontFamily="sans-serif-condensed-medium"
+                android:gravity="left"
+                android:textColor="@color/subtitle"
+                android:textSize="18sp"
+                app:layout_constraintStart_toEndOf="@+id/cart_product_image"
+                app:layout_constraintTop_toBottomOf="@+id/cart_product_name" />
+
+            <TextView
+                android:id="@+id/total_price_cart_product"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="16dp"
+                android:layout_marginRight="15dp"
+                android:layout_marginBottom="15dp"
+                android:fontFamily="@font/open_sans_condensed_bold"
+                android:gravity="right"
+                android:text="50$"
```

Comment:
Actually, our currency is **"TL"** not **"$"**

Reply:
it was just a mock data not the actual one